code/UI/Soundboard/Soundboard.razor

```diff
@@ -0,0 +1,79 @@
+@namespace Grubs.UI
+
+@if (Game.LocalPawn is not Grubs.Player player)
+{
+	return;
+}
+
+<root class="@(_isOpen ? "open" : "")">
+    <label class="sound-title" text=@_hoveredSound?.Title />
+	@{
+        var canPlaySound = GrubsConfig.SoundboardEnabled && player.TimeSinceSoundboardPlayed>= GrubsConfig.SoundboardCooldown;
+
+        <div class="items">
+		@foreach (var sound in GrubsGame.SoundboardSounds)
+		{
+				<SoundboardButton class="@(!canPlaySound ? "disabled" : "")"
+                Sound=@sound
+                @onmouseover=@(() => _hoveredSound = sound)
+                @onmouseout=@(() => _hoveredSound = null)
+                onclick=@( ()=> 
+				{
+					if( canPlaySound )
+						GrubsGame.PlaySoundboardSound(sound.Title); 
+				})></SoundboardButton>
+		}
+        </div>
+        
+        @* TODO: Add a slider for clientside soundboard volume when sliders get added. *@
+	}
+</root>
+
+@code {
+
+	private bool _isOpen;
+    private SoundboardSound _hoveredSound;
+
+	public override void Tick()
+	{
+		_isOpen = Input.Down(Grubs.InputAction.Soundboard) && !Inventory.IsOpen;
+        Cursor.Enabled("Soundboard", _isOpen);
+	}
+
+    protected override int BuildHash()
+    {
+        return HashCode.Combine(_isOpen, _hoveredSound?.GetHashCode());
```

Comment:
We need to include the hash code for when playing a sound is unavailable here. 
`var canPlaySound = GrubsConfig.SoundboardEnabled && player.TimeSinceSoundboardPlayed>= GrubsConfig.SoundboardCooldown;`